airbyte-integrations/connectors/source-google-webfonts/integration_tests/invalid_config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I believe this is an invalid one. But is it okay to include the one with an expired or invalid API key. Thanks.

Reply:
Done